--- addons/openerp/controllers/templates/footer.mako.orig	2011-08-03 08:42:46.282038064 +0000
+++ addons/openerp/controllers/templates/footer.mako	2011-08-03 08:43:28.330038043 +0000
@@ -3,6 +3,6 @@
     	${_("&copy; 2008-2010 %(ooweb)s  SA. All Rights Reserved ",
 		    ooweb="""<a target="_blank" href="http://openerp.com">OpenERP</a>""")|n}
         <br/>
-        ${_("Significant contributions to the web client have been made by")} <a target="_blank" href="http://axelor.com">Axelor</a>
+        <!-- ${_("Significant contributions to the web client have been made by")} <a target="_blank" href="http://axelor.com">Axelor</a> -->
     </div>
-</div>
\ No newline at end of file
+</div>
