--- addons/openerp/controllers/templates/login.mako.orig	2011-08-03 08:45:38.086042370 +0000
+++ addons/openerp/controllers/templates/login.mako	2011-08-03 08:49:54.194035332 +0000
@@ -23,7 +23,7 @@
     <table class="view" cellpadding="0" cellspacing="0" style="padding-top: 25px; border:none;">
         <tr>
             <td style="padding:35px 10px 5px 35px; min-width:100px;" valign="top" width="450">
-                <form action="${py.url(target)}" method="post" name="loginform" id="loginform" style="padding-bottom: 5px; min-width: 100px;">
+                <form action="${py.url(target)}" method="post" name="loginform" id="loginform" style="padding-bottom: 5px; min-width: 100px; width: 450px;">
                     % for key, value in origArgs.items():
                         <input type="hidden" name="${key}" value="${value}"/>
                     % endfor
@@ -78,7 +78,7 @@
                 % if info:
                     <div class="information">${info|n}</div>
                 % endif
-                <div style="margin-top: 10px">
+                <!-- <div style="margin-top: 10px">
                     <table cellpadding="0" cellspacing="0" width="100%" style="border:none;">
                         <tr>
                             <td style="padding-left:0;"><h3> ${_("Top Contributor:")}</h3></td>
@@ -87,10 +87,10 @@
                             <td style="padding-left:0;"><img src="/openerp/static/images/axelor_logo.png"/></td>
                         </tr>
                     </table>
-                </div>
+                </div> -->
             </td>
 
-            <td style="padding:55px 35px 5px 10px; min-width: 200px;" valign="top">
+            <!-- <td style="padding:55px 35px 5px 10px; min-width: 200px;" valign="top">
                 <p>${_("We think that daily job activities can be more intuitive, efficient, automated, .. and even fun.")}</p>
                 <h3>${_("OpenERP's vision to be:")}</h3>
 
@@ -124,7 +124,7 @@
                     </tr>
                 </table>
 
-            </td>
+            </td> -->
         </tr>
     </table>
     
